Resources/Info.plist

```diff
@@ -32,7 +32,7 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>1920</string>
+	<string>1</string>
```

Comment:
@Sherlouk shouldn't this read 892 if the script works?

Reply:
Nope, the whole point of this script is that Resources/Info.plist **never changes**!

The actual compiled version of the app (found in your derived data) will have a different Info.plist with the correct value.

Again, the whole point of this change is to stop merge conflicts from every branch having a different build version.